src/utils/getNumberOfFileChanges.ts

```diff
@@ -0,0 +1,27 @@
+import * as vscode from "vscode";
+import getFullBlame from "./getFullBlame";
+
+async function getNumberOfFileChanges(gitAPI:any, lineNumber:number) {
+    let blamePromises = await getFullBlame(
+        lineNumber,
+        lineNumber,
+        vscode.window.activeTextEditor?.document.uri.fsPath,
+        gitAPI
+    );
+
+    let blameArray: string[] = [];
+    let blames = await gitAPI?.repositories[0].blame(
+        vscode.window.activeTextEditor?.document.uri.fsPath || "."
+    );
+
+    blames.split("\n").forEach((line: String) => {
+        blameArray.push(line.split(" ")[0]);
+    });
+
+    let uniqueBlames = [
```

Comment:
Same question as before, why is this a Set?

Reply:
Same counter question

Reply:
Responded above